CONTRIBUTING.md

```diff
@@ -26,11 +26,11 @@ Then you can start working with the code :+1:
 
 <br>
 
-Do you want to **contribute a failing test**? [This tutorial will sow you how](https://github.com/rectorphp/rector/blob/main/docs/how_to_add_test_for_rector_rule.md)
+Do you want to **contribute a failing test**? [This tutorial will show you how](https://github.com/rectorphp/rector/blob/main/docs/how_to_add_test_for_rector_rule.md)
 
 ## Preparing Pull Request
 
-There 3 steps will make your pull-request easy to merge:
+4 steps will make your pull-request easy to merge:
```

Comment:
It's actually 3 steps. 
The coding standards are now outside the CI and could be removed.